README.md

```diff
@@ -9,7 +9,7 @@ urlFragment: microsoft-teams-apps-faqplus
 ---
 #  FAQ Plus App Template
 

```

Comment:
Should the links be changed too? Can you confirm if this really work

Reply:
Yes the links are changed and provided as per new URL.